terraform/key-vault.tf

```diff
@@ -0,0 +1,29 @@
+data "azurerm_client_config" "current" {}
```

Comment:
this is probably not required?

Reply:
I think it is, as the data object is needed in order to populate the tenant id for the key vault